### PR TITLE
adding assets folder back to website

### LINKED ISSUE #37: Implemente static serving of assets
We need to have some files to be served statically, eventually in the current path where we are doing it today, so that we do not break current demos and fiddle over the internet.

### PR DESCRIPTION
**Pull request by asturur**
In this way links for demos and old links around the internet should continue to work:

http://fabricjs.github.io/assets/dragon.jpg

should work after the merge

close #37

### CODE REVIEW

File: src/components/CarbonAd/CarbonAd.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const CarbonAd = () => {
     if (!ads || style.display === 'none') {
       setShowreminder(true);
       // eslint-disable-next-line no-undef, no-underscore-dangle
-    } else if (wind._carbonads) {
+    } else if (window._carbonads) {
```
Comment (asturur) [inline src/components/CarbonAd/CarbonAd.jsx:25]: error from previous PR

File: src/components/layoutFullWidth.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default function Layout({ children, darkBg }) {
     <>
       {/* <Seo></Seo> */}
       <Header />
-      <main className={darkBg && 'darkBg'}>
+      <main className={(darkBg ? 'darkBg' : '')}>
```
Comment (asturur) [inline src/components/layoutFullWidth.jsx:12]: React was complaining that the className was false

Comment (nimworks) [reply src/components/layoutFullWidth.jsx:12]: Ohh yeah! Expression evaluates to **false**! My bad 😇

**Comment by nimworks** [comment]
Now we can use http://fabricjs.github.io/assets/dragon.jpg in codePens as well 🤗

And we might only need to update it once we move to a custom domain. _(or it may still b available under the fabricjs.github.io domain)_

Other option is to dynamically fetch page domain using JS when fetching image and other assets within the CodePen

**Comment by asturur** [comment]
we will do a search replace once when i figure out how to move it ( have no idea, since the domain is not under my control, probably we have to move this under the other repo and overwrite the old one 🤷‍♂️  )

**Comment by asturur** [comment]
Yes this has been done mainly for the codepens.

**Comment by nimworks** [comment]
> probably we have to move this under the other repo and overwrite the old one

I guess just adding a the custom domain name to a CNAME file and then _(if not already added)_ updating DNS records with the DNS provider should do the trick!

We'd figure this out once we r at the point where this website could take over from the current website

**Comment by asturur** [comment]
Yes i do not have access to the dns for fabricjs.com, that is the issue.
I have to find the original owner.

**Pull request merged**